### DISCUSSION TITLE
"Translation text too long" with source string edits

### DISCUSSION THREAD

**Discussion by brockfanning** [WeblateOrg/weblate#5384] [Q&A] [answered]
Hello!

I'm hoping to figure out a way to work around a "Translation text too long" error. My impression is that normally this limit is automatically enforced as a way to improve translation quality, which is absolutely awesome. However I'm getting this error even when editing the source strings (in a monolingual setup). Does anyone know of a way to disable this limit for source string edits?

**Comment by nijel** [comment, marked as answer]
The limitation indeed doesn't make sense for the source strings, I've adjusted the calculation in 16415da5463b65a337b8289b8488d37d7353798e.

**Reply by brockfanning** [reply]
Wonderful, thank you!

**Comment by Shide** [comment]
This shouldn't be an error. It's a blocking behavior and should be threated as a Failed Check.
@nijel What should I do if the translation is correct? Rename the input string or what?

**Reply by nijel** [reply]
See https://docs.weblate.org/en/latest/user/translating.html#translated-string-length
